android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -92,12 +94,13 @@ protected NotificationCompat.Builder onExtendBuilder(@NonNull Context context, @
             builder.setPriority(PRIORITY_MAX);
         }
 
+        // Attempt to parse data and style notification
         if (message.containsKey(PAYLOAD_KEY)) {
             try {
                 JsonMap payload = JsonValue.parseString(message.getExtra(PAYLOAD_KEY)).optMap();
 
-                // Apply message style only for report comments
-                if (REPORT_COMMENT_TYPE.equals(payload.get(TYPE_KEY).getString())) {
+                // Apply message style when onyxData is present
```

Comment:
to clarify, this has to be Onyx data that we receive from Pusher? Urban Airship? just curious

Reply:
For Android/iOS we receive the onyxData update from both Pusher (only when the app is open) and Airship (we get the push notification regardless of whether the app is alive/killed). 

So in this code we only care that the notification data includes onyxData. I'll update the comment to make this a bit clearer.